src/Utils/Utils.php

```diff
@@ -160,6 +160,10 @@ public static function shouldHandleRequest(
         $context->addExtension('findologicService', $findologicService);
 
         $shopkey = $config->getShopkey();
+        if ($shopkey && trim($shopkey) === '') {
+            return $findologicService->disable();
+        }
```

Comment:
Based on the docblock it can be. Therefore I have added the falseness check.

![image](https://user-images.githubusercontent.com/20555112/102338450-27c71400-3f94-11eb-9827-94f49da6cf5d.png)

Reply:
As discussed, please disable on `null`.